sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/BaseSQLTests.java

```diff
@@ -79,6 +81,15 @@ public static void prepareTests() {
         testsToRun.clear();
     }
 
+    static File createInputScript(String... contents) throws IOException {
+        File result = File.createTempFile("script", ".sql", new File(rustDirectory));
+        result.deleteOnExit();
+        PrintWriter script = new PrintWriter(result, "UTF-8");
+        script.println(String.join(";\n", contents));
```

Comment:
System.getProperty("line.separator") instead of `\n` make the code more portable in case it needs to be run on windows.